### DISCUSSION TITLE
Is there need to define separate classes for both local and remote?

### DISCUSSION THREAD

**Discussion by ghost** [isar/isar#210] [Help] [answered]
Hello,

say I have a `User` from remote api with string id. Can I just create one class to represent both `User`s from api and from Isar local database, like this?

```
@Collection()
class User {

  @Id()
  int? localId;
 
  String? id;

  String firstName;

  String lastName;

}
```

Thanks

**Comment by simc** [comment, marked as answer]
Yes sure the code you posted is perfectly fine. I would also index the string id to make it unique and improve query performance if you search a specific string id.
